astrocut/tests/test_tess_footprint_cutout.py

```diff
@@ -77,6 +79,35 @@ def test_ffi_intersect(lon, lat, center, expected):
     assert intersection.value[0] == expected
 
 
+def test_ra_dec_crossmatch(coordinates, cutout_size):
+    all_ffis = get_ffis('s3://stpubdata/tess/public/footprints/tess_ffi_footprint_cache.json')
+
+    # Cutout size of 0 should do a point match
+    point_results = ra_dec_crossmatch(all_ffis, coordinates, 0, 21)
+    len_point_results = len(point_results)
+    assert isinstance(point_results, Table)
```

Comment:
I think it would be good to have an assertion that the crossmatch was actually a point search here. 

if you end up splitting up the crossmatch code into two separate function, it should be pretty easy to apply a mock/patch to those functions to assert that the crossmatch_point func was called and not the other.  maybe there are other ways to do it (does the returned table include any info we could check?)